src/main/java/org/hobbit/core/components/AbstractCommandReceivingComponent.java

```diff
@@ -64,17 +64,17 @@
     /**
      * Name of this Docker container.
      */
-    private String containerName;
+    protected String containerName;
```

Comment:
Why have all the visibility modifiers been changed? It does not make sense to me. Please explain why it is necessary.